manifest.js

```diff
@@ -6,9 +6,10 @@ import packageJson from './package.json' assert { type: 'json' };
  */
 const manifest = {
   manifest_version: 3,
-  name: packageJson.name,
+  default_locale: 'en',
+  name: '__MSG_extensionName__',
```

Comment:
How about adding the following comment to help users understand?

```suggestion
  /**
   * if you want to support multiple languages, you can use the following reference
   * https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization
   */
  name: '__MSG_extensionName__',
```

Reply:
Sorry for the delay @Jonghakseo & @PatrykKuniczak.
The change is pushed.